src/pages/index.js

```diff
@@ -38,21 +80,48 @@ const createCard = (cardData) => {
 const addCardObject = {
   popupSelector: "#add-card-modal",
   handleFormSubmit: (inputValues) => {
-    const cardElement = createCard(inputValues);
-    addCardForm.popupForm.reset();
-    addCardFormValidator.toggleButtonState();
-    cardSelection.addItem(cardElement);
-    addCardForm.close();
+    api.addCard(inputValues.name, inputValues.link).then((res) => {
+      // cardElement = createCard(inputValues);
+      // const cardElement = createCard(inputValues);
+      const cardElement = createCard(res);
+      addCardForm.popupForm.reset();
+      addCardFormValidator.toggleButtonState();
+      cardSelection.addItem(cardElement);
+      addCardForm.close();
+    });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**